The-Chonky-Panda/Atelier
# Style related products

## Description
Add styling to carousels to display according to visual design in business docs.

## How to test
manually - route to product that has more than 4 related products (e.g. 47423). In related products carousel, 4 product cards should display and a right arrow. click on right arrow, cards should shift over 1 to show the 5th card and hide the first one. Right arrow should hide and left arrow should appear. Clicking left arrow will shift back to original state.  if the main product has more than 5 related products, both left and right arrows should display if displaying the middle indexes. for the outfit carousel, add more than 3 outfits. right arrow should appear. clicking on right arrow should shift cards and hide right arrow if at the last card and display the left arrow. Clicking left arrow will shift cards back.

## Notes
Need to style modal so it appears in the related products area instead of covering the product detail at the top of the page.

## Issue tracking
https://trello.com/c/KrZsGIhG/141-l-fix-related-products-carousel-styling
https://trello.com/c/vH65f3qr/137-m-stop-add-product-card-from-jumping-position-when-adding-products-to-outfit
